src/components/project/dto/project.dto.ts

```diff
@@ -39,9 +39,9 @@ class Project extends Resource {
 
   @Field({
     description: 'The legacy department ID',
-    deprecationReason: 'Use `Project.departmentId` instead',
+    deprecationReason: 'Moved to the field resolver',
```

Comment:
Sorry for the confusion, I mean't the `deprecatedReason` should be removed here (`departmentId` is not deprecated) and added to the ResolveField's options for `deptId` which, still is, deprecated.